src/core/server/index.ts

```diff
@@ -298,6 +301,9 @@ export {
  */
 export interface RequestHandlerContext {
   core: {
+    auth: {
+      isAuthenticated: boolean;
+    };
```

Comment:
IMO, it feels more natural that this should be on the `KibanaRequest` rather than the context since it really describes a property of the request itself.

Reply:
ok, makes sense will update

Reply:
done